viz_scripts/generic_metrics.ipynb

```diff
@@ -119,193 +120,396 @@
+    "    generate_missing_plot(plot_title_no_quality,merged_debug_df,file_name)\n",
+    "    alt_text = store_alt_text_missing(merged_debug_df, file_name, plot_title_no_quality)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fc63a45",
+   "metadata": {},
+   "source": [
+    "### 2. Represents 80th percentile of the Number of Trips"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "about-seafood",
-   "metadata": {
-    "scrolled": false
-   },
+   "id": "77ece8ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_title_no_quality= \"Number of trips for 80th percentile for each mode (selected by users):\"\n",
+    "file_name = f'ntrips_under80{file_suffix}'\n",
+    "\n",
+    "try:\n",
+    "    df_confirmed_tc_u80 = pd.DataFrame()\n",
+    "    df_sensed_tc_u80 = pd.DataFrame()\n",
+    "    plot_title = plot_title_no_quality\n",
+    "    fig, ax = plt.subplots(nrows=2, ncols=1, figsize=(15,2*2), sharex=True)\n",
+    "\n",
+    "    if (expanded_ct.empty and expanded_ct_sensed.empty):\n",
+    "        missing_text = \"No data available for Confirmed Trips. \\n No data available for Sensed Trips.\"\n",
+    "    else:\n",
+    "        missing_text = \"\"\n",
+    "\n",
+    "    create_alt_html_title(plot_title, file_name, missing_text)\n",
+    "    create_alt_text_title(plot_title, file_name, missing_text)\n",
+    "\n",
+    "    if not expanded_ct.empty:\n",
+    "        df_confirmed_tc_u80_expanded, df_confirmed_tc_u80,cutoff, dist_threshold  = process_data_for_cutoff(expanded_ct,'Mode_confirm', distance_col, \"Labeled by user\\n (Confirmed trips)\")\n",
+    "        u80_quality_text = scaffolding.get_quality_text(expanded_ct, expanded_ct[expanded_ct['distance'] <= cutoff], \"<= \" + dist_threshold + \" \" + short_label, include_test_users)\n",
+    "        if not df_confirmed_tc_u80.empty:\n",
+    "            plot_title +=  \"\\n\" + \"For Labeled: \" + u80_quality_text\n",
+    "            alt_html = store_alt_html_stacked_bar_chart(df_confirmed_tc_u80_expanded, file_name)\n",
+    "            alt_text = store_alt_text_stacked_bar_chart(df_confirmed_tc_u80_expanded, file_name)\n",
+    "        else:\n",
+    "            print(\"df_confirmed_tc_u80 is empty.\")\n",
+    "    else:\n",
+    "        print(\"expanded_ct is empty.\")\n",
+    "\n",
+    "    if not expanded_ct_sensed.empty:\n",
+    "        df_sensed_tc_u80_expanded, df_sensed_tc_u80, cutoff_sensed, dist_threshold_sensed = process_data_for_cutoff(expanded_ct_sensed, 'primary_mode', distance_col, \"Sensed by OpenPATH\\n (Total trips)\")\n",
+    "        u80_quality_text_sensed = scaffolding.get_quality_text_sensed(expanded_ct_sensed[expanded_ct_sensed['distance'] <= cutoff_sensed], \"<= \" + dist_threshold_sensed + \" \" + short_label , include_test_users)\n",
+    "        if not df_sensed_tc_u80.empty:\n",
+    "            plot_title +=  \"\\n\" + \"For Sensed: \" + u80_quality_text_sensed\n",
```

Comment:
is overridden here

Reply:
The `plot_title` in this notebook code block is being appended with `plot_title += `...

Reply:
ah I see. Thanks for the clarification!